mercury-framework/mercury-grpc-service/src/main/java/com/redhat/mercury/resources/StringFormatter.java

```diff
@@ -0,0 +1,14 @@
+package com.redhat.mercury.resources;
```

Comment:
The resources package is more suited for Restful endpoints
```suggestion
package com.redhat.mercury.maven.grpc.utils;
```